docs/src/main/asciidoc/command-mode-reference.adoc

```diff
@@ -183,7 +195,7 @@ public class HelloTest {
     @Test
     @Launch("World")
     public void testLaunchCommand(LaunchResult result) {
-        Assertions.assertEquals("Hello World", result.getOutput());
+        Assertions.assertTrue(result.getOutput().contains("Hello World"));
```

Comment:
This is related to the JBoss Threads logging, right? I will fix it soon so if it's just that, keep both fixes out of the PR.

Reply:
I think, the full application logs are inside the `result;getOutput()` so you also have the Quarkus starting logs.
At least this is what @geoand confirmed, see https://quarkusio.zulipchat.com/#narrow/stream/187038-dev/topic/Command.20mode.20test

If you prefere, we can keep this PR opens untill you fix the JBoss Threads logging and see what it does.
I also plan to add an integration test for a Quarkus main so we can easily catch this kind of issue.

Reply:
Ah yes, but we should probably explain how to get rid of them in the guide. That would be better.

I personally use:
```
quarkus.log.level=SEVERE
quarkus.banner.enabled=false
```
in the `application.properties`.

Reply:
I'm not comfortable with this as usually sysout is not used but a real logger so a contains will always be needed due to logging format.

But if you prefere, I can keep the equals and add a note to explain that some configuration needs to be done for the test to success.

Reply:
@gsmet which solution do you prefere ? Keepin the contains or switch back to the equals and explaing the configuration to setup ?